pkg/query-service/app/server.go

```diff
@@ -142,6 +145,11 @@ func NewServer(serverOptions *ServerOptions) (*Server, error) {
 
 	s.privateHTTP = privateServer
 
+	_, err = opAmpModel.InitDB(constants.RELATIONAL_DATASOURCE_PATH)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
```suggestion
	if err := agentConf.Initiate(localDB, "sqlite"); err != nil {
		return nil, err
	}
```

Reply:
Won't this be a double server since the ee already starts one?

Reply:
This is just to initialise the Database, yeah agree that it will run twice in case of EE, but we need for the OSS version

Reply:
When ee runs this won't run.

Reply:
yes. This is separate from EE binary. 
EE binary flow -  ee/query-service/main.go > ee/query-service/app/server.go
OSS binary flow pkg/query-service/main.go > pkg/query-service/app/server.go